mitodl/mitxpro
# Various changes to register/login

#### Pre-Flight checklist


#### What are the relevant tickets?
Fixes #456 
Fixes #455 
Fixes #288 

#### What's this PR do?
This applies a few cosmetic changes to the login/register pages and move an exception case up into the common auth serializer so other endpoints catch it too.

#### How should this be manually tested?
Confirm that you can still login and register and the register details form changes are correct.


#### Screenshots (if appropriate)

<details>
  <summary>Screenshot of register form</summary>

![Screen Shot 2019-06-11 at 10 43 40](https://user-images.githubusercontent.com/28598/59284410-34014100-8c3a-11e9-8709-39a19b27af87.png)
![Screenshot_2019-06-11 mitxpro](https://user-images.githubusercontent.com/28598/59284360-22b83480-8c3a-11e9-83cf-c02159b8005e.png)
</details>
